_pkgdown.yml

```diff
@@ -19,6 +19,8 @@ navbar:
   left:
   - text: Custom themes
     href: articles/theming.html
+  - text: Sass variables
+    href: articles/available-sass-variables.html
```

Comment:
I think I'd prefer to default to calling these "theming variables", and then in the longer form writing, explain that theming vars are actually Sass variables (but only when Sass becomes relevant)

Reply:
That's much nicer, I agree.

Reply:
https://github.com/rstudio/bslib/pull/238/commits/bbcfb1b4b04286f5890c35e61a966a7a9f58b123